third_party/torch-mlir-dialects/lib/Dialect/TMTensor/IR/TMTensorOps.cpp

```diff
@@ -81,6 +82,233 @@ OpFoldResult TMTensor::getDim(OpBuilder &builder, Location loc, Value v,
   return builder.getI64IntegerAttr(t.getDimSize(dim));
 }
 
+
+//===----------------------------------------------------------------------===//
+// AttentionOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult AttentionOp::verify() {
+  Operation *op = getOperation();
+  ShapedType queryType = getQueryType();
+  ShapedType keyType = getKeyType();
+  ShapedType valueType = getValueType();
+  ShapedType outputType = getOutputType();
+  ArrayRef<int64_t> queryShape = queryType.getShape();
+  ArrayRef<int64_t> keyShape = keyType.getShape();
+  ArrayRef<int64_t> valueShape = valueType.getShape();
+  ArrayRef<int64_t> outputShape = outputType.getShape();
+  return success();
```

Comment:
The verifier here does not seem to match with what I see in torch-mlir: https://github.com/llvm/torch-mlir/blob/b9d29dc0550369b6e77c964b18234873d7099ef0/externals/llvm-external-projects/torch-mlir-dialects/lib/Dialect/TMTensor/IR/TMTensorOps.cpp#L91

Is that just skew between this PR and what has landed in Torch-MLIR?